backstop/tests/radio-tile.html

```diff
@@ -0,0 +1,610 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>Radio tile | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      section[id^='demo'] {
+        display: flex;
+        flex-direction: column;
+        gap: 11px;
+      }
+    </style>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/network"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/apple"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/android"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/windows"
+    ></script>
+    <script src="../assets/theme.js"></script>
+  </head>
+
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <h1>Radio tile</h1>
+    <hr />
+
+    <section id="demo-default">
+
+      <div class="iui-input-container">
+        <div class="iui-label iui-required">Radio tile label</div>
+
+        <div class="iui-radio-tile-container">
+          <input
```

Comment:
Not sure about this change.. Previously, we had `input` under `label` and thats how its done in react. Now if you move it out, `id` is required, which means we will need to generate that in react, if user does not pass it...
Any reason you kept it out?

Reply:
Moving `input` outside of `label` essentially reduced the specificity of radio tiles by `0, 1, 0` since everything in the radio tile was being referenced using the `> label` selector.

Reply:
amm, couldnt you skip `label` selector as you have separate classes?
I mean.. Im not sure we can go with this implementation in react for the reason I mentioned above.

Reply:
This has been updated. `<label>` is a parent of `<input>` ; therefore, `<input>` no longer needs an `id`

Check the updated PR description